src/common/hooks/useResolvePaymentType.ts

```diff
@@ -0,0 +1,18 @@
+/**
+ * Invoice Ninja (https://invoiceninja.com).
+ *
+ * @link https://github.com/invoiceninja/invoiceninja source repository
+ *
+ * @copyright Copyright (c) 2022. Invoice Ninja LLC (https://invoiceninja.com)
+ *
+ * @license https://www.elastic.co/licensing/elastic-license
+ */
+
+import { useStaticsQuery } from '../queries/statics';
+
+export const useResolvePaymentType = () => {
```

Comment:
`function` style of declaration, please.

Reply:
Changed!